components/Chips/src/MDCChipView.m

```diff
@@ -131,6 +132,7 @@ + (Class)layerClass {
 - (void)commonMDCChipViewInit {
   _minimumSize = kMDCChipMinimumSizeDefault;
   self.isAccessibilityElement = YES;
+  self.mdc_legacyFontScaling = YES;
```

Comment:
I believe this should be the _ivar_.

Reply:
Addressed in 1016545c1f8c1438528710bb7ac5d12436cb8a18.